### PR TITLE
Api 5040/appeals nod show/create

### PR DESCRIPTION
**Pull request by yakschuss**
[API-5040](https://vajira.max.gov/browse/API-5040)

This PR creates a sample environment for us to demonstrate and view created NODs. 

The behavior mimics that of HLR, and sets the state for upcoming oAuth changes.

### CODE REVIEW

File: samples/oauth_rails/VetsApiAppeals/app/views/nods/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+<script crossorigin src="https://unpkg.com/react-jsonschema-form/dist/react-jsonschema-form.js"></script>
+
+<script type="text/babel">
+  let headerSchema = <%= raw @header_schema %>;
+  let schema = <%=raw @schema %>;
+
+  // Fix react-jsonschema-form not interpreting some header defs correctly
+  for (const d in headerSchema['definitions']) {
+    if ( /^X-/.test(d) ) {
+      headerSchema['definitions'][d]['type'] = 'string';
```
Comment (cilestin) [inline samples/oauth_rails/VetsApiAppeals/app/views/nods/new.html.erb:20]: Is this necessary since you've included `type: string` in the schema?

Comment (yakschuss) [reply samples/oauth_rails/VetsApiAppeals/app/views/nods/new.html.erb:20]: I'm not sure! This was added by someone a whole lot more knowledgeable than I about the header-parser library used here ;) I can see what happens when I remove it.

Comment (yakschuss) [reply samples/oauth_rails/VetsApiAppeals/app/views/nods/new.html.erb]: Looks like the rendering was fine either way.

**Review by cilestin — APPROVED** [review]
LGTM!

Let me know in slack that you've merged it and I'll get it pushed up to heroku!

**Pull request merged**